README.md

```diff
@@ -76,6 +76,18 @@ OPTIONS:
 
 **Note:** You can specify the Apple ID email address and username as arguments when using the tool or by setting them as environment variables (`IPATOOL_EMAIL` and `IPATOOL_PASSWORD`). If you do not specify this information using either of those methods, the tool will prompt for user input in an interactive session. Similarly, you can supply the 2FA code interactively or using the environment variable `IPATOOL_2FA_CODE`.
 
+## Troubleshooting
+
+If `ipatool download` fails with the following error despite providing correct email/password/auth code:
+
+```
+==> ℹ️	[Info] Querying the iTunes Store for '<bundle-identifier>' in country '<country>'...
+==> ℹ️	[Info] Authenticating with the App Store...
+==> ❌	[Error] An unknown error has occurred.
+```
+
+then see [issue #30](https://github.com/majd/ipatool/issues/30) (particularly [this comment](https://github.com/majd/ipatool/issues/30#issuecomment-977852648)) for a fix. Note that you may need version 1.07.
+
```

Comment:
Let me know if you'd like this to be reworded or moved to a different section in the README.